cypress/integration/specialFeatures/cookieBanner/index.js

```diff
@@ -0,0 +1,51 @@
+import ramdaPath from 'ramda/src/path';
+import config from '../../../support/config/services';
+import getPaths from '../../../support/helpers/getPaths';
+import describeForEuOnly from '../../../support/helpers/describeForEuOnly';
+import visitPage from '../../../support/helpers/visitPage';
+import serviceHasPageType from '../../../support/helpers/serviceHasPageType';
+import runCanonicalTests from './testsForCanonicalOnly';
+import runAmpTests from './testsForAMPOnly';
+
+const serviceFilter = (service, pageType) =>
+  Cypress.env('SMOKE')
+    ? ['news', 'pashto'].includes(service)
+    : ramdaPath([service, 'pageTypes', pageType, 'smoke'], config);
```

Comment:
So when smoke testing we only test the cookie banners for `news` and `pashto` but otherwise we respect the `smoke` property in the Cypress config? Might this be a bit confusing as it makes the way`CYPRESS_SMOKE` works here different to other parts of the suite?

Is the problem that we want smoke testing of the banners to operate on a very reduced set of services, but the `smoke` property in the config doesn't let us define this at present (because it is only per-page type)?